src/etc/inc/util.inc

```diff
@@ -1850,6 +1850,16 @@ function format_bytes($bytes) {
 	}
 }
 
+function format_time($seconds) {
```

Comment:
Completely missed that, I found only format_bytes() and format_number() in utils.inc and looked for something similarly named. I'll fix the patch.